packages/lab/stories/navigation-item/navigation-item.doc.stories.mdx

```diff
@@ -16,81 +16,113 @@ import HelpAndSupport from "docs/blocks/help-and-support.doc.mdx";
 
 # Navigation Item
 
-A navigation item component is a repeatable UI element that facilitates the movement and exploration within a website or application. It typically consists of a group of links, buttons, or other interactive elements that allow users to navigate between different sections, pages, or features of the system.
+Navigation Item allows users to navigate between distinct applications and website pages.
+
+When the Navigation Item is not a parent, it contains a link, which users can click, and which receives focus via a keyboard interaction. When the Navigation Item is a parent, it will contain an expandable action, which contains a button instead.
+
+A Navigation Item hugs its content by default, but is customisable so that its container can be adjusted to fit any preferred width.
 
 ##### Use when
 
-- When your website or application requires deep navigation structures, implement multi-level navigation to allow users to orient themselves whilst navigating through different levels of content hierarchy or sections.
+- Use Navigation Item when you have multiple levels of navigation in your application or website.
 
-- When a sticky or fixed navigation option that is needed, which remains visible as users scroll down the page, always ensuring they have easy access to the website or applications main navigation.
+- We suggest first considering a horizontal group of Navigation Items for distinct webpages or applications. If you have multiple (more than 8) services to navigate to at a platform level, or have navigation across distinct apps, consider using a vertical group instead.
+
+- Our recommendation is to display Navigation Items in a fixed App Header (or header area) if you need your items to remain visible on scroll.
 
 ##### Don't use when
 
-- When content lacks clear categorization or hierarchy.
+- When you need to show different views of content within the same page, consider using [Tabs](?path=/docs/documentation-lab-tabs--page) instead.
+
+- When your content is very specific ,in this case, a minimalistic or single-page design may be more appropriate than a complex navigation structure. Consider using the [Interactable Card](?path=/docs/documentation-core-card--page), or a [Link](?path=/docs/documentation-core-link--page) component within the body of the page.
+
+## Content
+
+- Our recommendation is that all labels should be short and concise.
+
+- Always use labels in sentence case in a Navigation Item.
```

Comment:
Does this mean we need a different version for "Vertical group nested" example 

> Label 1 - Level 0

Reply:
Good spot! I'll update this